src/content/wrapped-ether.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Wrapped Ether (WETH)
+description: An introduction to Wrapped Ether (WETH)—an ERC20-compatible version of Ether (ETH). 
+---
+---
+# What Is Wrapped Ether (WETH)? {#what-is-wrapped-ether-weth}    
+Wrapped Ether (WETH) is an ERC-20 token representing Ether (ETH), Ethereum’s native coin. If you use Ethereum dapps, you've likely seen tokens built using the ERC-20 standard. The ERC-20 standard offers many benefits to developers and users, especially**flexibility** (tokens can be customized for different use cases) and **fungibility** (tokens are identical and interchangeable). 
+
+Ether (ETH) doesn't comply with the ERC-20 standard, but there's significant demand to use it in ERC20-compatible wallets and dapps. Wrapped Ether (WETH) solves the problem by providing an ERC20-compliant version of ETH to use in dapps or swap with other ERC-20 tokens.
+
+Today, **nearly 3% of the circulating ETH supply is locked in the WETH token contract**—inspiring its description as the "World's Most Popular Smart Contract". WETH is also one of the most used ERC-20 tokens, especially among users interacting with applications in Ethereum's Decentralized Finance (DeFi) ecosystem. 
+
+## Introduction to wrapped tokens {#intro-to-wrapped-tokens} 
+
+A wrapped token is a token whose value is pegged to an underlying asset. An amount of the original asset is deposited into a smart contract—similar to a digital bank vault—that mints an equal amount of wrapped tokens. The vault holds the deposit until you're ready to exchange your wrapped tokens for the original asset (so you can think think of a wrapped token as a redeemable IOU issued by a smart contract).
+
+Wrapping ETH today requires by depositing it in a smart contract which creates an amount of wrapped ETH (WETH) tokens equal to the original deposit. You can also “unwrap” ETH by sending WETH tokens to the contract and receiving ETH in return.
+
+In all cases, the conversion between WETH and ETH is always completed at a 1:1 ratio. Since the smart contract locks up ETH before minting WETH tokens, the value of WETH tokens in circulation is backed by ETH held in reserves. This keeps the prices of WETH and ETH relatively equal and ensures you can always swap both assets without realizing a loss. 
+
+## Why do we need to wrap ETH? {#why-do-we-need-to-wrap-eth} 
+
+Common use cases for wrapping tokens include extending the functionality of a token, allowing a native cryptocurrency to behave like an ERC20 token, or allowing the use of a token outside its native blockchain. The [canonical implementation of Wrapped Ether (WETH)](https://blog.0xproject.com/canonical-weth-a9aa7d0279dd) described here is designed for the former two use cases. 
```

Comment:
```suggestion
Common use cases for wrapping tokens include extending the functionality of a token, allowing a native cryptocurrency to behave like an ERC20 token, or allowing the use of a token outside its native blockchain.
```

Reply:
this sentence was highly confusing to me, bare in mind that the article already is difficult to read due to using lots of jargon like erc-20, tokes, smart contracts etc.

Reply:
I was heavy on emphasizing exactly what version of WETH we're talking about since there are so many versions of WETH out there. These tokens will have different use cases (eg. WETH on Polygon is to make ETH usable outside Ethereum) and security properties, so it seemed important to clarify. But I could simply add the link to that article in the "Why am I seeing different types of WETH?" FAQ

Reply:
I've also tweaked the paragraph a bit for more context (ie. "why is it important to have ETH behave like an ERC-20 token").

Reply:
> I was heavy on emphasizing exactly what version of WETH we're talking about since there are so many versions of WETH out there. These tokens will have different use cases (eg. WETH on Polygon is to make ETH usable outside Ethereum) and security properties, so it seemed important to clarify. But I could simply add the link to that article in the "Why am I seeing different types of WETH?" FAQ

hm that distinction was unclear to me. maybe that could be more obvious, explicitly stating that. either in faq or there

Reply:
There are a couple of places where I mentioned it. This is from the FAQs: 

> **Why am I seeing different types of WETH?**

> Besides the [canonical implementation of WETH](https://blog.0xproject.com/canonical-weth-a9aa7d0279dd) described om this page, there are other variants in the wild. These may be custom tokens created by dapp developers or versions issued on other blockchains and may behave differently or have different security properties. **Always double-check the token information to know which WETH implementation you're interacting with.**

And this is from the security section:

> Note that these security considerations apply only if you’re using WETH issued on Ethereum. [Blockchain bridges](https://github.com/ethereum/ethereum-org-website/blob/patch-34/bridges) make ETH usable on a non-Ethereum blockchain by issuing a variant of WETH compatible that blockchain’s technical standards.

> But this introduces additional security issues other than those described previously. For example, you must also believe the bridge is secure against attacks, the bridge operator(s) won’t freeze or steal your funds, etc.